ros_buildfarm/templates/snippet/builder_system-groovy_check-for-failing-jobs.xml.em

```diff
@@ -0,0 +1,82 @@
+@(SNIPPET(
+    'builder_system-groovy',
+    command=
+"""// CHECK FOR FAILING JOBS
+import hudson.model.Job
+
+// Control the number of failures in a row to check
+min_number_of_failures = 5
+// Control the minimun amount of time failing
+min_days_failing = 7
```

Comment:
I don't think we need to wait a full week. If it's been more than 2 days it shouldn't be an intermittent build farm outage.